config.md

```diff
@@ -79,6 +79,7 @@ For Solaris, the mount entry corresponds to the 'fs' resource in the [zonecfg(1M
 * **`options`** (array of strings, OPTIONAL) Mount options of the filesystem to be used.
   * Linux: supported options are listed in the [mount(8)][mount.8] man page. Note both [filesystem-independent][mount.8-filesystem-independent] and [filesystem-specific][mount.8-filesystem-specific] options are listed.
   * Solaris: corresponds to "options" of the fs resource in [zonecfg(1M)][zonecfg.1m].
+  * Windows: "ro" (readonly) is the only supported option for Windows.
```

Comment:
> Should we just change the wording to be the following?
>
> > Windows: runtimes MUST support ro, mounting the filesystem read-only when ro is given.

I'd [suggested backticks around the literal `ro`][1], but other than that [your line][2] looks like [my suggestion][1].  So yes, I think that's what you should change the wording to.

> If `ro` is not a great option, is there a better replacement we should consider?

I have no opinion on the value.  [`ro` is what Linux uses for this concept][3], but I'm fine with Windows using whatever it likes.

[1]: https://github.com/opencontainers/runtime-spec/pull/868#discussion_r119405474
[2]: https://github.com/opencontainers/runtime-spec/pull/868#discussion_r119449931
[3]:https://git.kernel.org/pub/scm/utils/util-linux/util-linux.git/tree/sys-utils/mount.8?h=v2.29.2#n1112